components/movementsensor/rtkutils/ntrip.go

```diff
@@ -31,6 +35,34 @@ type NtripConfig struct {
 	NtripUser            string `json:"ntrip_username,omitempty"`
 }
 
+// Sourcetable struct contains the stream.
+type Sourcetable struct {
```

Comment:
Does this need to be exported? Same with streams. Since they're in the same package.

Reply:
Not really. I will revert it.

Reply:
Actually we can't return unexportable struct in an exportable function.